Sources/PackageDescription/Target.swift

```diff
@@ -1462,7 +1462,7 @@ public extension PluginCommandIntent {
         return documentationGeneration
     }
 
-    /// The plugin formats source code.
+    /// The plug-in format's source code.
```

Comment:
This looks wrong, we mean the verb `format` here. A `sourceCodeFormatting` plugin can be used to format the user's code.

Reply:
Thanks Boris, I applied your change.